projects/dsp-ui/src/lib/viewer/services/value-type.service.ts

```diff
@@ -62,6 +62,20 @@ export class ValueTypeService {
         }
     }
 
+    /**
+     * Given the ObjectType of a PropertyDefinition, compares it to the type of the type of the provided value.
+     * Primarily used to check if a TextValue type is equal to one of the readonly strings in this class.
+     *
+     * @param objectType PropertyDefinition ObjectType
+     * @param valueType Value type (ReadValue, DeleteValue, BaseValue, etc.)
+     */
+    compareObjectTypeWithValueType(objectType: string, valueType: string): boolean {
+        return objectType === this._readTextValueAsString ||
```

Comment:
sorry for the late notice but I think there is some missing conditions:

```ts
(valueType === Constants.TextValue && objectType === this._readTextValueAsString) || ...
```

Otherwise text values are always included.

Reply:
fixed in https://github.com/dasch-swiss/dsp-ui-lib/pull/175